svm/src/transaction_processor.rs

```diff
@@ -1302,6 +1308,25 @@ mod tests {
                 .or_default()
                 .push((account, is_writable));
         }
+
+        fn calculate_fee(
```

Comment:
these are just for the tests. I assume it's fine if these get out of date?

Reply:
Yes, just for the tests